src/Symfony/Component/Console/Tests/ApplicationTest.php

```diff
@@ -50,6 +50,7 @@
 use Symfony\Component\EventDispatcher\EventDispatcherInterface;
 use Symfony\Component\EventDispatcher\EventSubscriberInterface;
 use Symfony\Component\Process\Process;
+use function xdebug_break;
```

Comment:
Symfony does not use `use function`

Reply:
it is also unused... I thought the CS fixer would take care of that?!